src/applications/personalization/profile-2/routes.js

```diff
@@ -47,6 +47,11 @@ const getRoutes = showDirectDeposit => {
       return route.component !== DirectDeposit;
     });
   }
+  if (!showMilitaryInformation) {
+    return routes.filter(route => {
```

Comment:
oh no, this isn't gonna work if `showDirectDeposit` is also false since the `if` right above this will return all the routes without DD but will still have military info

Reply:
Oh my goodness. Yeah this is wrong. Will fix 😞